torch/utils/data/sampler.py

```diff
@@ -51,3 +51,20 @@ def __iter__(self):
 
     def __len__(self):
         return self.num_samples
+
+
+class SubsetRandomSampler(Sampler):
+    """Samples elements randomly from a given list of indices, without replacement.
+
+    Arguments:
+        indices (list): a list of indices
+    """
+
+    def __init__(self, indices):
+        self.indices = indices
+
+    def __iter__(self):
+        return iter([self.indices[i] for i in torch.randperm(len(self.indices))])
```

Comment:
It seems that this didn't pass the check of python 2.7.8. 
Should I roll back to the previous version?